.travis.yml

```diff
@@ -39,11 +39,15 @@ script:
     [ -z "$lint" -o "$lint" = "Lint-free!" ]
   - go vet $(go list ./... 2>/dev/null | grep -v /vendor/)
   - go install github.com/keybase/kbfs/...
-  - cd libkbfs && go test -i && go test -race -c && ./libkbfs.test -test.timeout 4m
+  - cd libkbfs && go test -i && go test -race -c && ./libkbfs.test -test.timeout 1m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir ./libkbfs.test -test.timeout 1m
   - cd ../libfuse && go test -i && go test -c && ./libfuse.test -test.timeout 2m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir ./libfuse.test -test.timeout 2m
   - cd ../test
   - go test -i -tags fuse && go test -race -c && ./test.test -test.timeout 4m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir ./test.test -test.timeout 4m
   - go test -c -tags fuse && ./test.test -test.timeout 4m
+  - env KEYBASE_TEST_BSERVER_ADDR=tempdir go test -c -tags fuse && ./test.test -test.timeout 4m
```

Comment:
Overall, the new tests seems to have added about 80 seconds to the build.  I think it's probably overkill.  Specifically in this directory I don't think we need to run both of the libkbfs- and the fuse-based tests; the way they interact with the bserver should be exactly the same.  Can we eliminate one?